base_layer/wallet/src/utxo_scanner_service/utxo_scanner_task.rs

```diff
@@ -699,9 +700,10 @@ where
         client: &mut BaseNodeWalletRpcClient,
     ) -> Result<HeightHash, UtxoScannerError> {
         let birthday = self.resources.db.get_wallet_birthday()?;
-        // Calculate the unix epoch time of two days before the wallet birthday. This is to avoid any weird time zone
-        // issues
-        let epoch_time = u64::from(birthday.saturating_sub(2)) * 60 * 60 * 24;
+        // Calculate the unix epoch time of two days before the wallet birthday.
+        // This is to avoid any weird time zone issues
+        let epoch_time = get_birthday_from_unix_epoch(birthday, 2u16);
```

Comment:
I would go for 2 weeks, rather than 2 days. 
This is to go around re-org issues, we should rather error on the safe side/